plugins/example_plugin.bash

```diff
@@ -79,6 +79,8 @@ restore_data() {
   timestamp_day_dir=${timestamp_dir%??????}
   if [ -e "/tmp/GPBACKUP_PLUGIN_LOG_TO_STDERR" ] ; then
     echo 'Some plugin warning' >&2
+  elif [ -e "/tmp/GPBACKUP_PLUGIN_KILL_HELPER" ] ; then
+    pkill "gpbackup_helper"
```

Comment:
Why do we need a pkill if the test works fine with GPBACKUP_PLUGIN_DIE?
```diff
diff --git a/end_to_end/plugin_test.go b/end_to_end/plugin_test.go
index d5b9ce65..e00a0fd0 100644
--- a/end_to_end/plugin_test.go
+++ b/end_to_end/plugin_test.go
@@ -340,14 +340,14 @@ var _ = Describe("End to End plugin tests", func() {
 					"Instruct plugin to kill the helper",
 					cluster.ON_HOSTS,
 					func(contentID int) string {
-						return fmt.Sprintf("touch /tmp/GPBACKUP_PLUGIN_KILL_HELPER")
+						return fmt.Sprintf("touch /tmp/GPBACKUP_PLUGIN_DIE")
 					})
 
 				defer backupCluster.GenerateAndExecuteCommand(
 					"Unset plugin instruction",
 					cluster.ON_HOSTS,
 					func(contentID int) string {
-						return fmt.Sprintf("rm /tmp/GPBACKUP_PLUGIN_KILL_HELPER")
+						return fmt.Sprintf("rm /tmp/GPBACKUP_PLUGIN_DIE")
 					})
 
 				gprestoreCmd := exec.Command(gprestorePath,
```

Reply:
I previously tested with "--no-compression" - in this case simple early exit from plugin will not help to reproduce the problem.
But as the test currently doesn't use "--no-compression", I think we can use existing plugin code for `/tmp/GPBACKUP_PLUGIN_DIE`. 

Updated the code.